ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -33,10 +33,20 @@
 
 #include <string>
 
+/*
+ * In Azure: Buckets are synonymous with Containers, and Objects are synonymous with Blobs. This
+ * class represents an active connection to the Azure endpoint and allows for interaction with the
+ * Azure client. The azure_connection allows for the following functionality: Listing the container
+ * contents filtered by a given prefix and output all or output single, puts a blob to the cloud,
+ * gets a blob from the cloud, and deletes a blob from the cloud. It also can check for the
+ * existance of a unique container or blob. Each azure_connection is associated with a unique
```

Comment:
"existance" is still spelt incorrectly - thanks for fixing the rest of it!

Reply:
fixed :) thanks for picking this up